data/gegno/engines.txt

```diff
@@ -0,0 +1,225 @@
+outfit "RG15 Torch Thruster"
+	category Engines
+	licenses
+		"Gegno Civilian"
+	cost 26000
+	mass 31
+	thumbnail "outfit/small torch"
+	"engine capacity" -31
+	"outfit space" -31
+	thrust 7.67
+	"thrusting energy" 0.9
+	"thrusting heat" 2
+	"flare sprite" "effect/ion torch/small"
+		"frame rate" 2.5
+	"flare sound" "ion torch small"
+	description "The RG15 Ion Torch Thruster is the smallest torch thruster found amongst Gegno Vi ships and it is almost exclusively mounted on the Slate. It's not amazingly efficient, weighing more and producing more heat than most thrusters of a similar size, but also has a slightly lower energy consumption rate."
+
+outfit "RG15 Torch Steering"
+	category Engines
+	licenses
+		"Gegno Civilian"
+	cost 21000
+	mass 25
+	thumbnail "outfit/small torch steer"
+	"engine capacity" -25
+	"outfit space" -25
+	turn 205
+	"turning energy" 0.4
+	"turning heat" 1.4
+	"steering flare sprite" "effect/ion torch/small"
+		"frame rate" 2.4
+	"steering flare sound" "ion torch small"
+	description "This steering system that incorporates RG15 thrusters is only meant for the smallest of Gegno Vi ships, and mounting it on anything larger than a Shuttle-class ship is unheard of. Despite this, its energy requirement is a notch lower than normal, so some Gegno pilots use it as a cheap addon."
+
+outfit "RG18 Torch Thruster"
+	category Engines
+	licenses
+		"Gegno Civilian"
+	cost 58000
+	mass 51
+	thumbnail "outfit/med torch"
+	"engine capacity" -51
+	"outfit space" -51
+	thrust 14.8
+	"thrusting energy" 1.2
```

Comment:
Is this a bit low? Currently, the RG15 does ~8.52 thrust/energy, the RG3 does ~9.16, this value puts the RG18s at ~12.3.
They also don't really lose anything elsewhere:
![image](https://user-images.githubusercontent.com/20605679/173209813-ea8682a6-51ab-4bc5-aca3-3933b87a509b.png)
The thrust/ton and thrust/heat are both inbetween the other two, the thrust/heat might possibly be a bit close to the RG3s.
Maybe somewhere between 1.63 and 1.73 for the energy on these ones? 1.73 gives thrust/energy of ~8.55, 1.63 gives ~9.08. I like 1.68 which gives 8.81.

Reply:
The idea between the differences is to show how massive the RG3 is for its size compared to the other two as it only powers 3 of the largest ships in Gegno space. It is *supposed* to be farther ahead in values, but I suppose if were building upon an almost equal factor between engine sizes, this would make more sense. What do you say?

Reply:
What I'm trying to say is, the RG3 was built to be that large. It is actually the oldest of the engines, as it's based from their city-ships in older times. The RG15 and RG18 are newer attempts to condense this technology, but the Vi have problems doing so the smaller the tech goes, since they've mainly worked large-scale (think computer technology). So, basically, the RG3 is not the "step up" from the step-up (RG18) by any means.